src/SprykerEco/Yves/Billpay/Plugin/ServiceProvider/TwigBillpayServiceProvider.php

```diff
@@ -0,0 +1,256 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Billpay\Plugin\ServiceProvider;
+
+use Generated\Shared\Transfer\CountryTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Silex\Application;
+use Silex\ServiceProviderInterface;
+use Spryker\Shared\Config\Config;
+use Spryker\Shared\Kernel\Store;
+use Spryker\Yves\Kernel\AbstractPlugin;
+use SprykerEco\Shared\Billpay\BillpaySharedConfig;
+use SprykerEco\Shared\Billpay\BillpayConstants;
+use Twig_SimpleFunction;
+
+/**
+ * @method \SprykerEco\Yves\Billpay\BillpayFactory getFactory()
+ */
+class TwigBillpayServiceProvider extends AbstractPlugin implements ServiceProviderInterface
+{
+    const MONEY_DIVIDER = 100;
+
+    /**
+     * @param \Silex\Application $app
```

Comment:
Those - based on phpstan - will most likely have to be annotated as spryker ones.